From b746ae5d6cfeb62b870eb117a99de41a64480c0d Mon Sep 17 00:00:00 2001
From: Jerome Brunet <jbrunet@baylibre.com>
Date: Sat, 28 Mar 2020 21:56:04 +0100
Subject: [PATCH 1/2] meson: env: remove amlogic directory from fdtfile
 variable

The "amlogic" directory with fdtfile is messing with yocto extlinux
default configuration. It does not seems very useful and I'm to lazy
to look at extlinux right now

Signed-off-by: Jerome Brunet <jbrunet@baylibre.com>
Upstream-Status: Inappropriate [oe specific]
---
 include/configs/meson64.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/meson64.h b/include/configs/meson64.h
index efab9a624dc5..0c06ea756919 100644
--- a/include/configs/meson64.h
+++ b/include/configs/meson64.h
@@ -133,7 +133,7 @@
 	"pxefile_addr_r=" PXEFILE_ADDR_R "\0" \
 	"fdtoverlay_addr_r=" FDTOVERLAY_ADDR_R "\0" \
 	"ramdisk_addr_r=" RAMDISK_ADDR_R "\0" \
-	"fdtfile=amlogic/" CONFIG_DEFAULT_DEVICE_TREE ".dtb\0" \
+	"fdtfile=" CONFIG_DEFAULT_DEVICE_TREE ".dtb\0" \
 	"dfu_alt_info=fitimage ram " KERNEL_ADDR_R " 0x4000000 \0" \
 	BOOTENV
 #endif
-- 
2.43.0

